net-p2p/bitcoin-core/bitcoin-core-25.1-r1.ebuild

```diff
@@ -144,23 +181,39 @@ src_configure() {
 		--enable-util-tx
 		--${wallet}-util-wallet
 		--disable-util-util
+		# syscall sandbox is missing faccessat2 and pselect6, causing bitcoind to crash during tests;
+		# removed upstream for 26.0 in https://github.com/bitcoin/bitcoin/commit/32e2ffc39374f61bb2435da507f285459985df9e
+		--without-seccomp
 		$(use_with libs)
 		$(use_with daemon)
-		$(use_with gui)
+		$(use_with gui gui qt5)
 		$(use_with dbus qtdbus)
 		$(use_with system-leveldb)
 		$(use_with system-libsecp256k1)
 	)
 	econf "${myeconfargs[@]}"
 }
 
+src_test() {
+	emake check
+
+	use daemon && edo "${PYTHON}" test/functional/test_runner.py \
```

Comment:
Personal opinion, that you may take or leave, but I like the verbose style `if use daemon; then…` better if the if-block is a long line.